rss2017/process/commentProcess.js

```diff
@@ -0,0 +1,34 @@
+functionTrack(){
+    var or3client = lib.or3client;
+
+    console.log('the Public Comment Process is here');
+    var origNote = or3client.or3request(or3client.notesUrl+'?id='+note.forum, {}, 'GET', token);
+
+    var conference = 'RSS 2017 Workshop '+TRACK_NAME+' Track'
+
+    origNote.then(function(result) {
+      var forum = result.notes[0];
+      var note_number = forum.number;
+
+      var reviewers = [PAPERGRP + '/Reviewers'];
+      var authors = forum.content.authorids;
+
+      var author_mail = {
+        "groups": authors,
+        "subject": "Review of your submission to "+TRACK+": \"" + forum.content.title + "\"",
+        "message": "Your submission to " + conference + " has received a public comment.\n\n
+            Title: " + note.content.title + "\n\nReview: " + note.content.comment + "\n\n
+            To view the review, click here: " + baseUrl+"/forum?id=" + note.forum
+      };
+
+      var authorMailP = or3client.or3request( or3client.mailUrl, author_mail, 'POST', token );
+
+      return Promise.all([
```

Comment:
if there is only one promise you don't need to use Promise.all just return the single promise